openmetadata-ui/src/main/resources/ui/src/components/common/UserTeamSelectableList/UserTeamSelectableList.component.tsx

```diff
@@ -181,13 +231,10 @@ export const UserTeamSelectableList = ({
     setCount((pre) => ({ ...pre, team: res.data.hits.total.value }));
   };
 
-  const fetchCount = async () => {
+  const init = async () => {
     if (popupVisible) {
-      if (owner?.type === EntityType.USER) {
-        await getTeamCount();
-      } else {
-        await getUserCount();
-      }
+      await Promise.all([getUserCount(), getTeamCount()]);
```

Comment:
We were calling once API because other count can be derived from the Actual list API call, Avoiding 2 calls